test/xspec-param-as_stylesheet.xspec

```diff
@@ -4,9 +4,9 @@
 	xmlns:myv="http://example.org/ns/my/variable"
 	xmlns:x="http://www.jenitennison.com/xslt/xspec">
 
-	<x:scenario label="In context template-param (i.e. //x:context/x:param),">
+	<x:scenario label="In context template-param (i.e. //x:context/x:param), when @as uses a prefix defined in x:param,">
 		<x:context mode="mirror:param-mirror">
-			<x:param as="xs:double" name="param-items" select="xs:integer(0)" />
+			<x:param as="local-xs:double" name="param-items" select="xs:integer(0)" xmlns:local-xs="http://www.w3.org/2001/XMLSchema" />

```

Comment:
Is there any value in changing `xspec-param-as.xspec` to define the `local-xs` namespace prefix on `x:description` and then using this prefix in some of the `@as` attributes? It might be tangential, but you can decide.

Reply:
Good point.
If `x:description` declares the `local-xs` namespace, the bug is not reproduced. That's what "not widely available" in my initial [description](https://github.com/xspec/xspec/pull/894#issue-396906140) means:

> if `@as` has namespace prefix which is not widely available.

I should have commented it in the test to clarify that.
I added the comment in c285a501d0d70a40a5fdbc311a7fe61ad2cc3493.

Reply:
OK, so adding the declaration on `x:description` is tangential to the bug fix you're making here. The extra comment looks good.